packages/config/config/devices/0x031e/lzw60.json

```
@@ -0,0 +1,205 @@
+			"valueSize": 1,
+			"unit": "%",
+			"minValue": 10,
+			"maxValue": 50,
+			"defaultValue": 10,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"12": {
+			"label": "PIR Sensitivity",
+			"description": "Change the sensitivity of the PIR (motion) sensor; 0 = off, 1 (low) - 10 (high)",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 10,
+			"defaultValue": 10,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"13": {
+			"label": "PIR Trigger Time (Time Between PIR Readings)",
+			"valueSize": 2,
+			"unit": "seconds",
+			"minValue": 5,
+			"maxValue": 15300,
+			"defaultValue": 30,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"14": {
+			"label": "Basic Set Command Send After PIR Trigger",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				},
+				{
+					"label": "Enabled",
+					"value": 1
+				}
+			]
+		},
+		"15": {
+			"label": "Reverse Basic Set Behavior in Association Group 2",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Send on when triggered, off when clear",
+					"value": 0
+				},
+				{
+					"label": "Send off when triggered, on when clear",
+					"value": 1
+				}
+			]
+		},
+		"101": {
+			"label": "Temperature Reporting Interval",
+			"description": "Note: The set value will round to the nearest minute.",
+			"valueSize": 4,
+			"unit": "seconds",
+			"minValue": 0,
+			"maxValue": 2678400,
+			"defaultValue": 7200,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"102": {
+			"label": "Humidity Reporting Interval",
+			"description": "Note: The set value will round to the nearest minute.",
+			"valueSize": 4,
+			"unit": "seconds",
+			"minValue": 0,
+			"maxValue": 2678400,
+			"defaultValue": 7200,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"103": {
+			"label": "Luminance Reporting Interval",
+			"description": "Note: The set value will round to the nearest minute.",
+			"valueSize": 4,
+			"unit": "seconds",
+			"minValue": 0,
+			"maxValue": 2678400,
+			"defaultValue": 7200,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"104": {
+			"label": "Battery Reporting Interval",
+			"description": "Note: The set value will round to the nearest minute.",
+			"valueSize": 4,
+			"unit": "seconds",
+			"minValue": 0,
+			"maxValue": 2678400,
+			"defaultValue": 7200,
```

Manual says default is 86400
```suggestion
			"defaultValue": 86400,
```